telegram/poll.py

```diff
@@ -39,11 +39,11 @@ class PollOption(TelegramObject):
     considered equal, if their :attr:`text` and :attr:`voter_count` are equal.
 
     Attributes:
-        text (:obj:`str`): Option text, 1-100 characters.
+        text (:obj:`str`): Option text, 1-300 characters.
         voter_count (:obj:`int`): Number of users that voted for this option.
 
     Args:
-        text (:obj:`str`): Option text, 1-100 characters.
+        text (:obj:`str`): Option text, 1-300 characters.
```

Comment:
Option text is still 100 characters, as per bot api docs

Reply:
oh thats right I changed the wrong strings thanks!